mgmtsystem/ec2.py

```diff
@@ -16,6 +21,12 @@
 )
 
 
+def regions(regionlist, regionname):
```

Comment:
no, all modules have the regions function, so all you need is to pass in a module